spec/definitions.yaml

```
@@ -982,6 +982,15 @@ definitions:
               increase the likelihood to get instances for this pool, especially spot instances at
               a low rate. If false, only the exact type set as `instance_type` is used.
               Added with AWS release v11.2.0.
+      azure:
+        type: object
```

This can be `null`, right? If correct, then we should add it to the `aws` field, as well

```suggestion
        type: object
        x-nullable: true
```

---

Oh, yes. Thanks! Good catch!